eval/builtin_fn_misc.go

```diff
@@ -60,6 +63,19 @@ func constantly(args ...interface{}) Callable {
 	)
 }
 
+func getenv(fm *Frame, key string) error {
```

Comment:
This function doesn't need to take a `Frame`, it can return `value`. Non-error return values are converted to output.

Reply:
Yeah, this makes sense